app/views/homes/index.html.erb

```diff
@@ -1 +1 @@
-<%= react_component("Home", {}, class: 'min-h-screen bg-gray-200') %>
+<%= react_component("Home", {}, class: 'min-h-screen bg-gray-100', style: 'height: calc(100vh - 4.5rem - 1.5rem); overflow-y: hidden;') %>
```

Comment:
can u explain why u do this?

Reply:
This is so that the page has no scroll, because now everything fits, at any screen size or zoom level.
I substract 4.5rem that is the NavBar height, and 1.5rem, the top and bottom margins

Reply:
I have changed this styles for a class called full-page-height located in application.tailwind.css